src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,99 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  font-weight: 400;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: $product-card-max-width;
+  padding: 32px 16px 16px;
+  margin-right: $product-card-margin-right;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__name {
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $dark-grey;
+  }
+
+  &__code {
+    align-self: flex-start;
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $light-grey;
+  }
+
+  &__rating {
+    display: flex;
+    align-self: stretch;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+  }
+
+  &__review {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+  }
+
+  &__price {
+    display: flex;
+    align-self: stretch;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-title {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $light-grey;
+  }
+
+  &__price-cost {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__buy {
+    width: 100%;
+    height: 40px;
```

Comment:
You've already set paddings, just make a block from this link and no need to hardcode it's height